cmd/juju/ssh/debugcode_test.go

```diff
@@ -22,7 +22,7 @@ type DebugCodeSuite struct {
 
 func (s *DebugCodeSuite) TestArgFormatting(c *gc.C) {
 	s.setupModel(c)
-	s.setHostChecker(validAddresses("0.public"))
+	s.setHostChecker(validAddresses(22, "0.public"))
```

Comment:
To avoid the churn, we could keep `validAddresses()` as is and have it set the accepted port to 22.
Then introduce a new method `validaAddressesWithPort()` and call it from the newly added test.
Then eliminates the changes in all the existing tests just to pass in port 22.